tasks/run-matlab-command/v0/main.ts

```diff
@@ -10,14 +10,16 @@ import { architecture, platform } from "./utils";
 async function run() {
     try {
         taskLib.setResourcePath(path.join( __dirname, "task.json"));
-        const command = taskLib.getInput("command", true);
-        await runCommand(command as string);
+        const command: string = taskLib.getInput("command", true) || "";
+        const startupOpts: string = taskLib.getInput("startupOptions") || "";
```

Comment:
I feel like we're ending up pretty much back where you started, but maybe:
```typescript
const startupOptions = taskLib.getInput("startupOptions");
await runCommand(command, startupOpts);
...

async function runCommand(command: string, args?: string) {
...
if (args) {
    runTool.arg(args.split(" "));
}
...
```
The output for taskLib.getInput will implicitly have type `string | undefined` (can add the annotation if we want to be explicit) so we can pass it as an optional argument, which seems easier to understand than providing `""` as a required argument.

Sorry to basically take you back in a circle to where we started 😄

Reply:
No worries!

Reply:
Fixed